onnxruntime/core/common/threadpool.cc

```diff
@@ -46,9 +46,7 @@ ThreadPoolProfiler::ThreadPoolProfiler(int num_threads, const CHAR_TYPE* thread_
   child_thread_stats_.assign(num_threads, {});
   if (thread_pool_name) {
 #ifdef _WIN32
-    using convert_type = std::codecvt_utf8<wchar_t>;
-    std::wstring_convert<convert_type, wchar_t> converter;
-    thread_pool_name_ = converter.to_bytes(thread_pool_name);
+    thread_pool_name_ = ToUTF8String(thread_pool_name);
```

Comment:
It's not your fault, but thread_pool_name_ has no reason to be a UTF-8 string. You can't print out a UTF-8 string to screen on Windows. You can't pass out thread_pool_name_ to printf(...).

Reply:
Should it have been in the local codepage?

Reply:
I think so.